libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -270,6 +270,7 @@ bool ASTJsonConverter::visit(ContractDefinition const& _node)
 		make_pair("abstract", _node.abstract()),
 		make_pair("baseContracts", toJson(_node.baseContracts())),
 		make_pair("contractDependencies", getContainerIds(_node.annotation().contractDependencies, true)),
+		make_pair("emittedEvents", getContainerIds(_node.interfaceEvents(true))),
```

Comment:
Why does this belong to the AST btw?

Reply:
It is important information like the `contractDependencies` above. The AST with its annotations contains much more than jus the syntactical information.

Reply:
I think it might be because the call graph annotations are not available if you stop after parsing and don't run the analysis.

Reply:
Oh, the reason is actually different: https://github.com/ethereum/solidity/pull/10996#issuecomment-786026787